ecl/dedicated_hypervisor/v1/vcenter.py

```diff
@@ -68,3 +71,18 @@ def update(self, session, vcenter_id, password, license_id):
         )
         self._translate_response(resp, has_body=True)
         return self
+
+
+class LinkLocalAddresses(resource2.Resource):
+    base_path = '/vcenters/addresses'
+    resources_key = 'addresses'
+    resource_key = 'address'
```

Comment:
```suggestion
    resource_key = 'address'
    service = dedicated_hypervisor_service.DedicatedHypervisorService()
```
すみません、serviceの設定も必要でした。